### PR TITLE
project-chatbot Eliane

### PR DESCRIPTION
**Pull request by El1an3**
## Netlify link
https://deluxe-sunshine-7a2b80-chatbot.netlify.app/

## Collaborators
none

### CODE REVIEW

File: .DS_Store
Comment (NellaMori) [inline .DS_Store:1]: .DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file (if you don't have one, you need to create it) 👍

File: code/.DS_Store
Comment (NellaMori) [inline code/.DS_Store:1]: same here, delete please 🧹

File: code/assets/.DS_Store
Comment (NellaMori) [inline code/assets/.DS_Store:1]: same here, delete please 🧹

File: code/assets/bot-2.jpg
Comment (NellaMori) [inline code/assets/bot-2.jpg:1]: if you haven't used this image, go ahead and delete it 👍

File: code/assets/user-2.jpg
Comment (NellaMori) [inline code/assets/user-2.jpg:1]: same for this image 👍

File: code/script.js
Comment (NellaMori) [inline code/script.js:1]: Please remove the console.log() around the code. Generally they need to be deleted when the code has been tested and works as meant to 🧹

**Review by AntonellaMorittu — APPROVED** [review]
Hi Eliane, good job with your chatbot! 🤖 I've found out I am a Mandarin fish apparently 😆 The code is well structured, you meet the requirements and you also put into practice the skills learnt so far, amazing! ⭐  To improve even more the UX you could add validation for the name field so that is not empty for example?

I left some comments specifically on some parts, please take a look at them :) Thank you!

**Pull request closed without merging**